src/Juvix/Compiler/Core/Language.hs

```diff
@@ -18,55 +18,87 @@ import Juvix.Compiler.Core.Language.Base
 {---------------------------------------------------------------------------------}
 {- Program tree datatype -}
 
+-- De Bruijn index of a locally bound variable.
```

Comment:
please, use Haddock comments to document types and functions.
e.g.  
```
-- | De Bruijn index of a locally bound variable.
```
Only the first line of the comment needs the `|`

Reply:
Okay, sorry, I forgot.

Reply:
Done